src/main/java/com/cburch/logisim/std/bfh/BfhLibrary.java

```diff
@@ -35,14 +35,14 @@
 import com.cburch.logisim.tools.Tool;
 import java.util.List;
 
-public class BFHPraktika extends Library {
+public class BfhLibrary extends Library {
   /**
    * Unique identifier of the library, used as reference in project files.
    * Do NOT change as it will prevent project files from loading.
    *
    * Identifier value must MUST be unique string among all libraries.
    */
-  public static final String _ID = "BFH-Praktika";
+  public static final String _ID = "BFH";
```

Comment:
Value of library `_ID` needs to stay unaltered despite the refactoring.

Reply:
My bad, i missed those to e8902a4.